renovate.json

```diff
@@ -130,8 +131,38 @@
       "at any time"
     ]
   },
+  "gomod": {
+    "enabled": true,
+    "additionalBranchPrefix": "{{baseBranch}}/",
+    "branchPrefix": "konflux/mintmaker/",
+    "postUpdateOptions": ["gomodTidy", "gomodUpdateImportPaths"],
+    "postUpgradeTasks": {
+      "commands": ["go mod vendor"],
+      "fileFilters": ["vendor/**"]
+    },
+    "packageRules": [
+      {
+        "groupName": "Go dependencies",
+        "commitMessageAction": "",
+        "commitMessageTopic": "Go dependencies",
+        "matchManagers": ["gomod"],
+        "excludePackageNames": [
+          "k8s.io/kubernetes"
```

Comment:
need to also exclude OpenShift ones, they are handled in the submodule update script

https://github.com/openshift/windows-machine-config-operator/blob/master/hack/update_submodules.sh#L37